querybook/webapp/components/DataDocGitHub/GitHub.scss

```diff
@@ -6,3 +6,56 @@
 .GitHubAuth-icon {
     margin-bottom: 20px;
 }
+
```

Comment:
try to use the css variables for the margin/padding/border-radius/etc if possible